ui-core/src/main/res/drawable/edit_text_color_selector.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Few things that can be improved here:

1. This file is not really a drawable, but actually a color. It should be located in ui-core/src/main/res/color
2. A better name would be `text_color_primary`
3. The disabled color is not nicely customisable and it forces a certain color that might not match the actual text color used. I'd suggest doing something like this:
```XML
<selector xmlns:android="http://schemas.android.com/apk/res/android">
    <!-- 0.32 is the alpha value that material design uses for disabled text -->
    <item android:alpha="0.32" android:color="@color/textColor" android:state_enabled="false" />
    <item android:color="@color/textColor" />
</selector>
```